pom.xml

```diff
@@ -90,37 +89,85 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <version>2.5</version>
+      <artifactId>subversion</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <version>1.1</version>
+      <artifactId>scm-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <version>1.10</version>
+      <artifactId>ssh-credentials</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <version>1.16.1</version>
+      <artifactId>credentials</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <version>1.18</version>
+      <artifactId>script-security</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <version>${pipeline.version}</version>
+      <artifactId>workflow-support</artifactId>
+      <version>2.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <version>2.1</version>
       <artifactId>workflow-step-api</artifactId>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-cps</artifactId>
+      <version>2.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-job</artifactId>
-      <version>${pipeline.version}</version>
+      <version>2.1</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-aggregator</artifactId>

```

Comment:
Unfortunately they must be added

The issue with [JENKINS-21486](https://issues.jenkins-ci.org/browse/JENKINS-21486) is that starting with core 2.16, Jenkins is more "strict" when loading plugins which causes. When running the tests against core versions greater than that (or when the baseline is >= 2.16) the tests fails as the new validations are not satisfied.

This happens because when running the tests the Maven dependency resolution "wins", which provide a dependency graph that it incorrect from the Jenkins perspective.

The solution (as it has been applied in many Pipeline plugins) is to bring the dependencies to the first level instead of let Maven bring them transitively. I.e., all the added dependencies here were either transitive or implied (detached plugins). The reason to change `workflow-aggregator` for individual components is to reduce the "transitive set" to analyze.